src/main/java/com/minecolonies/coremod/colony/permissions/Permissions.java

```diff
@@ -474,6 +474,22 @@ public boolean setOwner(final PlayerEntity player)
         return true;
     }
 
+    /**
+     * Change the owner of a colony to [abandoned]
+     */
+    @Override
+    public void setOwnerAbandoned()
+    {
+        players.remove(getOwner());
+
+        ownerName = "[abandoned]";
+        ownerUUID = new UUID(235123534,131231234);
+
```

Comment:
There is a UUID function to get a random UUID

Reply:
oh that one is indeed nicer, changed that